core/src/test/scala/com/github/johnynek/paiges/PaigesTest.scala

```diff
@@ -361,11 +360,16 @@ the spaces""")
     }
   }
 
-  test("if deunioned is a subset, then isSubDocOf") {
+  test("if isSubDocOf then deunioned is a subset") {
     forAll { (a: Doc, b: Doc) =>
       val da = a.deunioned
       val db = b.deunioned
-      assert(a.isSubDocOf(b) == SortedSet(da: _*).subsetOf(SortedSet(db: _*)))
+      if (a.isSubDocOf(b)) {
+        assert(SortedSet(da: _*).subsetOf(SortedSet(db: _*)))
+      }
+      else {
+        ()
```

Comment:
In these cases I'd rather either leave off the "else branch" or else use `else succeed`. This seems like the worst of both words to me.